static/library/bar_component.ts

```diff
@@ -80,6 +88,11 @@ export class DatacommonsBarComponent extends LitElement {
   @property({ type: Array<string>, converter: convertArrayAttribute })
   comparisonVariables;
 
+  // Optional: Whether to render as a lollipop
+  // Set to true to render using lollipops instead of bars
+  @property()
```

Comment:
change to `@property({type: Boolean})` - otherwise i think Lit will actually make this a string (even though it's typed as a boolean)

Reply:
Thanks for the pointer! I think this is why I had to set lollipop="true" with the string above originally. Fixed.